test/unit/dialog-service.spec.js

```diff
@@ -227,4 +227,49 @@ describe('the Dialog Service', () => {
         done();
       });
   });
+
+  it('".open" properly propagates errors', (done) => {
+    let catchWasCalled = false;
+    let promise = dialogService.open({ viewModel: 'test/fixtures/non-existent' })
+      .catch(() => {
+        catchWasCalled = true;
+      }).then(() => {
+        expect(catchWasCalled).toBe(true);
+        done();
+      });
+  });
+
+  it('reports no active dialog after ".closeAll" has been invoked', (done) => {
+    const settings = { viewModel: TestElement };
+    const dialogsToOpen = 6;
+    let i = dialogsToOpen;
+    let controllersPromises = [];
+    let controllers;
+    let catchWasCalled = false;
+
+    expect(dialogService.hasActiveDialog).toBe(false);
```

Comment:
@PWKad  I'd like to clear the behavior of `.closeAll()` first, then I'll add more tests in own `describe`. Can you check my first comment?